// [[Reason: TODO: Import type has different module attributes]] ////

//// [tests/cases/conformance/node/nodeModulesImportTypeModeDeclarationEmit1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,7 +1,7 @@
 
 
 //// [/.src/out/index.d.ts]
 export type LocalInterface = import("pkg", { assert: { "resolution-mode": "require" } }).RequireInterface & import("pkg", { assert: { "resolution-mode": "import" } }).ImportInterface;
-export declare const a: import("pkg").RequireInterface;
-export declare const b: import("pkg", { with: { "resolution-mode": "import" } }).ImportInterface;
+export declare const a: import("pkg", { assert: { "resolution-mode": "require" } }).RequireInterface;
+export declare const b: import("pkg", { assert: { "resolution-mode": "import" } }).ImportInterface;
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
